src/components/edit/Edit.js

```diff
@@ -0,0 +1,255 @@
+import React from 'react'
+import axios from 'axios';
+import { useState } from 'react';
+import { useParams } from 'react-router-dom';
+
+
+// const url = `http://localhost:4000/v1//agent/property/ + id`;
+
+
+const Edit = () => {
+    console.log(useParams())
+    const {id} = useParams()
+
+    console.log(id)
+    const token = localStorage.getItem("token");
+    console.log(token);
+    let config = {
+      headers: {
+        Authorization: token,
+        "content-type": "multipart/form-data",
+      },
+    };
+    const [inputProperties, setInputProperties] = useState({
+      image_url: "",
+      title: "",
+      price: "",
+      status: "",
+      address: "",
+      city: "",
+      lga: "",
+      zip_code: "",
+      neighbourhood: "",
+    //   country: "",
+      number_of_beds: "",
+      number_of_baths: "",
+      land_size: "",
+    });
+    // const [file, setFile] = useState();
+    // const saveImage = (e) => {
+    //   setFile(e.target.files[0]);
+    // };
+    const handleChange = (e) => {
+      setInputProperties({ ...inputProperties, [e.target.name]: e.target.value });
+      // console.log(inputProperties);
+    };
+    const handleSubmit = async (e) => {
+      e.preventDefault();
+      // const formData = new FormData();
+  
+      // formData.append("file", file);
+      // formData.append("propertyInfo", JSON.stringify(inputProperties));
+      try {
+        const response = await axios.put(`http://localhost:4000/v1/agent/property/${id}`, inputProperties, config);
```

Comment:
You shouldn't hardcode your API URL. It should be dynamic based on the environment you are on. When you deploy this code online, your API URL will no longer be localhost:3006, how will you solve the issue in that case?